### DISCUSSION TITLE
HTTP ERROR 502

### DISCUSSION THREAD

**Discussion by arunkuttytry** [postalserver/postal#2110] [Help with using Postal]
after install Caddy i am facing HTTP Error 502

How to fix

![image](https://user-images.githubusercontent.com/107031529/184157944-be6c5087-2878-49ac-aa87-3ae81f52a84e.png)

**Comment by willpower232** [comment]
A quick internet search confirms that HTTP 502 is gateway not found which implies that whilst your Caddy is running, it is either misconfigured or Postal is not running

**Comment by bonest** [comment]
I found my postal server down today as well. 502 when I tried to login on the webinterface.  I did an apt upgrade yesterday and it may have stopped postal.

ssh into the server to see that postal was not running, which you can see by issuing the 'postal status' command

I restarted the server by running 'postal start' and the problem was fixed.
